<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeighborJoinWidget</class>
 <widget class="QWidget" name="NeighborJoinWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>236</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>525</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="distanceMatrixTab">
      <attribute name="title">
       <string>Distance Matrix</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="modelLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="lblModel">
           <property name="text">
            <string>Distance matrix model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbModel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="coefficientLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="lblCoefficient">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Coefficient of variation of substitution rate among sites</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="sbAlpha">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="chbGamma">
           <property name="text">
            <string>Gamma distributed rates across sites</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="transitionLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="lblTransition">
           <property name="text">
            <string>Transition/transversion ratio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="transitionRatioSpinBox">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bootstrapTab">
      <attribute name="title">
       <string>Bootstrapping and Consensus Tree</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QCheckBox" name="chbEnableBootstrapping">
         <property name="text">
          <string>Enable bootstrapping</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="bootstrapOptionsContainer" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="replicatesNumberLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="lblReplicatesNumber">
              <property name="text">
               <string>Number of replicates</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbReplicatesNumber">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="seedLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="lblSeed">
              <property name="text">
               <string>Seed (must be odd)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbSeed">
              <property name="maximum">
               <number>32765</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
              <property name="value">
               <number>99</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="consensusTypeLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="lblConsensusType">
              <property name="text">
               <string>Consensus type</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbConsensusType"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="fractionLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="lblFraction">
              <property name="text">
               <string>Fraction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbFraction">
              <property name="minimum">
               <double>0.500000000000000</double>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="displayOptionsTab">
      <attribute name="title">
       <string>Display Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="U2::PhyTreeDisplayOptionsWidget" name="displayOptions" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>162</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>U2::PhyTreeDisplayOptionsWidget</class>
   <extends>QWidget</extends>
   <header location="global">U2View/PhyTreeDisplayOptionsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chbGamma</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbAlpha</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbGamma</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblCoefficient</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbEnableBootstrapping</sender>
   <signal>toggled(bool)</signal>
   <receiver>bootstrapOptionsContainer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
